packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -1,174 +1,136 @@
 <template>
-  <div
-    class="sf-image"
-    :class="{ 'has-size': size }"
-    :style="size"
-    v-on="$listeners"
-  >
-    <template v-if="(isSrcset && isSrcsetArray) || isSrcObject">
-      <picture>
-        <source
-          v-for="(srcItem, i) in getSrcsetWhenIsArray.srcset"
-          :key="i"
-          :srcset="srcItem.src"
-          :media="srcItem.media"
-          :type="srcItem.type"
-        />
-        <img
-          v-show="getSrcsetWhenIsArray.src"
-          :src="getSrcsetWhenIsArray.src"
-          v-bind="$attrs"
-          :width="width"
-          :height="height"
-        />
-      </picture>
-    </template>
-    <template v-else>
-      <img
-        v-show="getSrcOrSrcset"
-        v-bind="{ ...$attrs, ...getSrcOrSrcset }"
-        :width="width"
-        :height="height"
-      />
-    </template>
-    <noscript v-if="noscript" inline-template>
-      <img
-        class="noscript"
-        :src="noscript"
-        v-bind="$attrs"
-        :width="width"
-        :height="height"
-      />
-    </noscript>
-    <div v-if="hasOverlay" class="sf-image__overlay">
+  <div class="sf-image--wrapper" :style="variables">
```

Comment:
It should be, since if user adds them specifically. However, we leave the options to override using CSS directly on the img tag, so instead of adding to `img`, we modify the CSS variables on its wrapper.